CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # App Center SDK for Android Change Log
 
-## Version 1.10.1 (Under active development)
+## Version 1.10.1
```

Comment:
This update has a feature, so the new version should be `1.11.0`

Reply:
Sure. we will change the version number.